src/decisionengine_modules/glideinwms/publishers/fe_group_classads.py

```diff
@@ -50,13 +49,19 @@ def create_invalidate_constraint(self, requests_df):
                 self.invalidate_ads_constraint[collector_host] = constraint
 
     def dataframe_for_entrytype(self, allow_type, datablock):
-        requests_df = datablock.get("glideclient_manifests")
         facts_df = datablock.get("de_logicengine_facts")
 
-        data_product_name = f"Factory_Entries_{allow_type}"
+        requests_df = datablock.get("glideclient_manifests")
+        if requests_df.empty:
+            return pandas.DataFrame()
+
```

Comment:
Not a blocker, but is it fairly trivial to add a test to make sure that if the `glideclient_manifests` data block is empty, we get an empty `pandas.DataFrame()`?

Reply:
Yes, I'll find a way to cover this.